src/fiatExchanges/SelectProvider.tsx

```diff
@@ -453,7 +453,9 @@ function ExchangesSection({
       ...analyticsData,
     })
     if (exchangesLink === SelectProviderExchangesLink.ExchangeQRScreen) {
-      navigate(Screens.ExchangeQR, { flow, exchanges })
+      navigate(Screens.QRNavigator, {
+        screen: Screens.QRCode,
+      })
```

Comment:
Is this change ok'd by product? I didn't see it in the linear issue, but seems like the title of the screen changes from Deposit Crypto to the My Code / Scan tab switcher.

Reply:
I'll clarify